semantic_model_generator/snowflake_utils/env_vars.py

```diff
@@ -1,5 +1,8 @@
 import os
 
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
Is this a no-op if the `.env` is absent?

Reply:
Yep!